cross-country/example.gpx

```diff
@@ -0,0 +1,109 @@
+				</svg>
+				]]></cc:notes>
+    </extensions>
+  </wpt>
+
+	<!-- Tulip and Notes drawings -->
+  <wpt lat="0" lon="0">
+    <extensions>
+      <cc:distance>0.6</cc:distance>
+			<!-- Full SVG -->
+      <cc:tulip><![CDATA[
+				<svg>
+				  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
+				</svg>
+				]]></cc:tulip>
+			<cc:notes><![CDATA[
+				<svg>
+				  <text>SVG Example</text>
+				</svg>
+				]]></cc:notes>
+    </extensions>
+  </wpt>
+
+  <!-- or any other tags (using them all at once is nonsensical) -->
+  <wpt lat="0" lon="0">
+    <extensions>
+      <cc:distance>0.35</cc:distance>
+      <cc:wpc clear="20" name="ABC"/>
+      <cc:wpm open="800" clear="200" name="XYZ"/>
+      <cc:wps open="800" clear="20" name="K99"/>
+      <cc:wpe clear="200"/>
+      <cc:dss/>
+      <cc:checkpoint/>
+      <cc:ass/>
+      <cc:timecontrol/>
+      <cc:cap>360</cc:cap>
+    </extensions>
```

Comment:
Does it make sense to try to prevent authors from creating nonsensical waypoints like this one? I assume it would require we add a new complexType with restrictions maybe?

Reply:
Hard call. I'm interested in growing adoption more than enforcing logic. At the moment, anyway ;-)